src/Workspaces/Core/Desktop/Workspace/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -653,7 +653,9 @@ private class ResolvedReferences
                 {
                     var documentationService = _workspace.Services.GetService<IDocumentationProviderService>();
                     var docProvider = documentationService.GetDocumentationProvider(outputFilePath);
+#pragma warning disable CA2000 // Dispose objects before losing scope - metadata instance wrapped within the returned MetadataReference instance.
```

Comment:
:memo: This is a repeat of an issue above; it should be treated as a leak and not as a transfer of ownership.